src/office-toolbox.ts

```diff
@@ -231,9 +231,10 @@ async function sideload(application: string, manifestPath: string) {
       application = await promptForApplication();
     }
 
-    if(application === 'word' || application === 'excel' || application === 'powerpoint'){
+    if(util.applicationProperties[application].canSideload){
```

Comment:
`util.applicationProperties[application]` is done over and over. Consider putting this into a variable and referencing that:

```

const appProperties = util.applicationProperties[application];


if (appProperties.canSideload) {

```